ssl/t1_lib.c

```diff
@@ -788,7 +792,8 @@ static const uint16_t tls_default_sigalg[] = {
     TLSEXT_SIGALG_ecdsa_sha1, /* SSL_PKEY_ECC */
     TLSEXT_SIGALG_gostr34102001_gostr3411, /* SSL_PKEY_GOST01 */
     TLSEXT_SIGALG_gostr34102012_256_gostr34112012_256, /* SSL_PKEY_GOST12_256 */
-    TLSEXT_SIGALG_gostr34102012_512_gostr34112012_512 /* SSL_PKEY_GOST12_512 */
+    TLSEXT_SIGALG_gostr34102012_512_gostr34112012_512, /* SSL_PKEY_GOST12_512 */
+    0 /* SSL_PKEY_ED25519 */
```

Comment:
It's not only in the "signature_algorithms" extension but also the "Supported Elliptic Curves" extension. All previous 25 curves did not require distinct "signature_algorithms" entries after all.

I find nothing in the document that would restrict EdDSA to TLS >= 1.2. Otherwise I guess the introduction would not put ECDSA and EdDSA at the same level.

And EdDSA is explicitely mentioned as applicable before TLS 1.2 in section 5.10:

> [...] Such standardization never took place, and as a result, SHA-1 is used in
> TLS 1.1 and earlier (except for EdDSA, which uses identity function).